cmd/sovereignnode/chainSimulator/tests/bridge/incoming_test.go

```diff
@@ -0,0 +1,176 @@
+package bridge
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"math/big"
+	"testing"
+	"time"
+
+	chainSim "github.com/multiversx/mx-chain-go/integrationTests/chainSimulator"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/components/api"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/configs"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/process"
+	sovereignChainSimulator "github.com/multiversx/mx-chain-go/sovereignnode/chainSimulator"
+	"github.com/multiversx/mx-chain-go/sovereignnode/dataCodec"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	coreAPI "github.com/multiversx/mx-chain-core-go/data/api"
+	"github.com/multiversx/mx-chain-core-go/data/block"
+	"github.com/multiversx/mx-chain-core-go/data/sovereign"
+	"github.com/multiversx/mx-chain-core-go/data/transaction"
+	"github.com/multiversx/mx-sdk-abi-incubator/golang/abi"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	eventIDDepositIncomingTransfer = "deposit"
+	topicIDDepositIncomingTransfer = "deposit"
+	hashSize                       = 32
+	token                          = "TKN-123456"
+	amountToTransfer               = "123"
```

Comment:
Let's please not have these here.
Use functions which can be reused to take these values as input parameters.
This is related to this comment: https://github.com/multiversx/mx-chain-go/pull/6213#discussion_r1618799021